### ISSUE TITLE
User Avatar img is missing alt text

### ISSUE THREAD

**Issue by kimberlythegeek** [mozilla-lockbox/lockbox-extension#557]
All images must have alt text to describe their purpose to screenreader users.

Example: `<img alt="User Avatar"` or `"User Profile Picture" src="default-avatar.svg">`

**Comment by Codexpanda** [comment]
Hi,

I would like to get contribute to this bug.

**Comment by devinreams** [comment]
Thanks for your interest, @Codexpanda. As described in our [contribution guide](https://github.com/mozilla-lockbox/lockbox-extension/blob/master/docs/contributing.md#how-to-contribute-code), it will be helpful to first understand the work you think needs to happen before submitting any patches (pull requests).

If you haven't already, here's how I would start thinking about it:

- inventory the use of avatar images in the code (just one place, many?)
- review how they're output in the app (consistently, differently?)
- any changes needed to the code beyond adding the alt text (underlying react changes needed?)

I hope this is fairly straight-forward but please let us know if you have questions as you think about it and consider the code changes. Thanks again!

**Comment by Codexpanda** [comment]
Hi @devinreams 

Sorry for very late reply. I checked the code and found where img tag is used. shared detailed below.

- for ./webextension/list/manage/components/intro-page.js   will add **tag id** in ./webextension/locales/en-US/list.ftl
- for ./webextension/list/manage/components/homepage.js   will add **tag id** in  ./webextension/locales/en-US/list.ftl
- for ./webextension/firstrun/components/app.js will add **tag id** in ./webextension/locales/en-US/firstrun.ftl
- for ./webextension/unlock/components/app.js  will add **tag id** in ./webextension/locales/en-US/unlock.ftl

I have a doubt
There are other two places where img tag is like below

- ./webextension/widgets/panel.js:            img src={imgSrc} alt="go bACk"
- ./webextension/list/manage/components/account-summary.js:          img alt="uSEr aVATAr" src={avatar}

here alt=value  the text value is not in proper case, feeling its killing read-ability, can i change this to  **snake_case** format and maintain same everywhere ?

**Comment by jimporter** [comment]
> for ./webextension/list/manage/components/intro-page.js will add tag id in ./webextension/locales/en-US/list.ftl

This already has a title attribute, which should be sufficient for accessibility.

> for ./webextension/list/manage/components/homepage.js will add tag id in ./webextension/locales/en-US/list.ftl

**Comment by jimporter** [comment]
Also, I think we resolved this issue in #709 as part of getting our a11y tests passing. Does this sound right, @devinreams?

**Comment by devinreams** [comment]
Thanks for taking a look, @Codexpanda. I hope @jimporter's explanations above make sense, we _should_ be OK with what we have at this time and appreciate your help double checking the code to confirm.

If you have some clever alt text ideas for those presentation-only images then please feel free to submit a pull request with those and ask an accessibility expert to help confirm they make sense.

For now I think this issue is OK to close. Thanks again 🙇
